5_lesson/lesson5-1.py

```diff
@@ -0,0 +1,15 @@
+"""
+Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+
+*Пример:*
+
+A = 3; B = 5 -> 243 (3⁵)
+    A = 2; B = 3 -> 8
+    """
+def stage(a, b):
+    if b>1:
+        return stage(a, b - 1) * a
+    else: return a
+print(stage(int(input("Введите число",)), int(input("Введите степень числа",))))
```

Comment:
Руслан, не забывайте форматировать код по PEP8.
```suggestion
def stage(a, b):
    if b > 1:
        return stage(a, b - 1) * a

    return a


print(stage(int(input("Введите число", )), int(input("Введите степень числа", ))))
```

Reply:
Что-то не так
![image](https://user-images.githubusercontent.com/49741194/224532530-b0074d41-70e6-4cc0-bceb-c1d755750800.png)